api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -420,6 +424,13 @@ message Watchdog {
   type.v3.Percent multikill_threshold = 5;
 }
 
+// Fatal actions to run while crashing.
+// We will run all safe actions before we run unsafe actions.
+message FatalAction {
+  // Extension specific configuration for the action.
```

Comment:
@envoyproxy/api-shepherds per discussion earlier today, should we have some declaration of the interface that extensions must conform to? The problem with that in general is that this might be different in Envoy/gRPC, but in this specific case, this is a bootstrap and Envoy-only extension point, so I think that would be pretty useful.

Reply:
Discussed this a bit out of band with Harvey. 

The comments for the extension now point to the interface it's expected to confirm to, and where extensions should live.

Thoughts @envoyproxy/api-shepherds ? Thanks!